ipyvue/VueWidget.py

```diff
@@ -1,24 +1,97 @@
-from traitlets import (Unicode, Instance, Union, List, Any, Dict)
-from ipywidgets import DOMWidget
-from ipywidgets.widgets.widget import (widget_serialization, CallbackDispatcher)
+from traitlets import HasTraits, Unicode, Instance, Union, List, Any, Dict
+from ipywidgets import DOMWidget, link
+from ipywidgets.widgets.widget import widget_serialization, CallbackDispatcher
 from ._version import semver
 from .ForceLoad import force_load_instance
 
 
+class ClassList:
+    def __init__(self, obj):
+        self.obj = obj
+
+    def remove(self, *classes):
+        """
+        Remove class elements from the class_ trait of the linked object.
+        Use coma separated str class names.
+        """
+
+        classes = [str(c) for c in classes]
+
+        src_classes = self.obj.class_.split() if self.obj.class_ else []
+        dst_classes = [c for c in src_classes if c not in classes]
+
+        self.obj.class_ = " ".join(dst_classes)
+
+        return
+
+    def add(self, *classes):
+        """
+        add class elements to the class_ trait of the linked object.
+        Use coma separated str class names.
```

Comment:
"comma separated" gives me the impression it's about strings instead of parameters. I'm not sure what's the best way to describe this in python docs. Maybe somtheing like this:
```
    add class elements to the class_ trait of the linked object.

    :param *classes (str): The classes to remove
```

Reply:
This line can now be removed